docs/docs/specs/v0/ui-system.md

```diff
@@ -0,0 +1,41 @@
+# UI System
+
+Here is how the UI system is organized into separate concerns:
+
+## UI Primitives
+> Lookless, pure components.
+@dxos/react-{…}
+packages/ui/primitives/react-{…}
+
+## Aurora UI
+> Look-ful, pure components.
+@dxos/aurora
+packages/ui/aurora
+
+> theme elements and tokens that can be applied to primitives
+@dxos/aurora-theme
+packages/ui/aurora-theme
+
+## Aurora X
+> Other big pieces that are congruent with Aurora and separately packaged. e.g.: aurora-composer, aurora-datagrid.
+> These are always provided in pure form, and with an "optional" ECHO adapter if necessary.
+@dxos/aurora-* 
+packages/ui/aurora-*
+
+## Shell
+> HALO Button and shell panels and popovers
+@dxos/react-shell
+packages/sdk/react-shell
+
+## Shared 1P libs:
+> Error boundaries and telemetry harnesses, logos and other visual and non-visual things shared between 1P apps
+@braneframe/react-appkit
+packages/ui/react-appkit
+
+## Braneframe (Composer v2)
+@braneframe/app
+packages/apps/braneframe-app
+
+## Braneframe plugins
+@braneframe/plugin-*
+packages/plugins/*
```

Comment:
I think I may have said that plugins should be a top-level packages dir. After reading the composer extensibility spec I'm feeling less sure about that because I think they're composer-specific.

Reply:
aye i'm there with you